test/tests/outputs/png/wedge.i

```diff
@@ -0,0 +1,82 @@
+[Mesh]
+  file = wedge.e
+  uniform_refine = 1
+[]
+
+[Functions]
+  active = 'tr_x tr_y'
+
+  [./tr_x]
+    type = ParsedFunction
+    value = -x
+  [../]
+
+  [./tr_y]
+    type = ParsedFunction
+    value = y
+  [../]
+[]
+
+[Variables]
+  active = 'u'
+
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  active = 'diff forcing dot'
+
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+
+  [./forcing]
+    type = GaussContForcing
+    variable = u
+    x_center = -0.5
+    y_center = 3.0
+    x_spread = 0.2
+    y_spread = 0.2
+  [../]
+
+  [./dot]
+    type = TimeDerivative
+    variable = u
+  [../]
+[]
+
+[BCs]
+  #active = ' '
+
+  [./Periodic]
+    [./x]
+      primary = 1
+      secondary = 2
+      transform_func = 'tr_x tr_y'
+      inv_transform_func = 'tr_x tr_y'
+    [../]
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  dt = 0.5
+  num_steps = 6
+  solve_type = NEWTON
+[]
+
+[Outputs]
+  [png]
+    type = PNGOutput
+    PNGFile = "PNGTEST"
+    resolution = 250
+    inColor = false
+  []
+  execute_on = 'timestep_end'
+  file_base = out_wedge
```

Comment:
could we just agree not to use `file_base` in tests for no reason. It is so much simpler if the output files follow the automatic moose convention (and the test names as well for that matter). Otherwise you always have to consult the input file to know which outout belongs to which test.

Reply:
I do agree - I actively discourage it. Going back and fixing up all the existing tests will take time.